test/basic.js

```diff
@@ -50,7 +50,7 @@ test('fails if from doesnt exist', t => {
 test('fails if mkdir fails', t => {
   var from = path.resolve(fixtures, 'from.env')
   var to = path.resolve(fixtures, 'from.env/a/b/c')
-  return t.rejects(cmdShim(from, to), { code: /^(ENOTDIR|EEXIST)$/ })
+  return t.rejects(cmdShim(from, to), { code: /^(ENOTDIR|EEXIST|ENOENT)$/ })
```

Comment:
I needed to add this to get the tests to pass on Windows. Let me know if I should rever this.